tensorflow/python/keras/losses.py

```diff
@@ -1693,16 +1693,16 @@ def cosine_similarity(y_true, y_pred, axis=-1):
 
   Usage:
 
-  >>> y_true = [[0., 1.], [1., 1.]]
-  >>> y_pred =[[1., 0.], [1., 1.]]
+  >>> y_true = [[0., 1.], [1., 1.], [1., 1.]]
+  >>> y_pred = [[1., 0.], [1., 1.], [-1., -1.]]
   >>> loss = tf.keras.losses.cosine_similarity(y_true, y_pred, axis=1)
   >>> # l2_norm(y_true) = [[0., 1.], [1./1.414], 1./1.414]]]
   >>> # l2_norm(y_pred) = [[1., 0.], [1./1.414], 1./1.414]]]
   >>> # l2_norm(y_true) . l2_norm(y_pred) = [[0., 0.], [0.5, 0.5]]
   >>> # loss = -sum(l2_norm(y_true) . l2_norm(y_pred), axis=1)
   >>> #       = -[0. + 0., 0.5 + 0.5]
   >>> loss.numpy()
```

Comment:
@mihaimaruseac I've edited the `y_true` and `y_pred` but i couldn't get above lines. I tried getting l2_norms of `y_true` and `y_pred` using `nn.l2_normalize(y_true, axis=1)` and `nn.l2_normalize(y_pred, axis=1)`. I got following output: 
```
tf.Tensor(
[[0.         1.        ]
 [0.70710677 0.70710677]
 [0.70710677 0.70710677]], shape=(3, 2), dtype=float32) tf.Tensor(
[[ 1.          0.        ]
 [ 0.70710677  0.70710677]
 [-0.70710677 -0.70710677]], shape=(3, 2), dtype=float32)
```

Reply:
1/1.414 is 1/sqrt(2) is 0.707

I would remove the comment lines